content/webapp/components/Exhibition/Exhibition.tsx

```diff
@@ -80,19 +77,17 @@ function getadmissionObject(): ExhibitionItem {
 function getTodaysHoursObject(): ExhibitionItem {
   const todaysHoursText = 'Galleries open Tuesday–Sunday, Opening times';
 
-  const link = {
+  const link: prismicT.RTLinkNode = {
     type: 'hyperlink',
-    start: todaysHoursText.length - 13,
+    start: todaysHoursText.length - 'Opening times'.length,
```

Comment:
Trying to make it clearer where the `13` comes from.

Also, we can just declare this as an instance of `prismicT.RTLinkNode`; we don't need to cast.